tests/console/force_scheduled_tasks.pm

```diff
@@ -39,10 +40,10 @@ sub run {
     my $before = time;
     # run cron jobs or systemd timers which can affect system performance and mask systemd timers later
     assert_script_run('find /etc/cron.{hourly,daily,weekly,monthly} -type f -executable -exec echo cron job: {} \; -exec {} \;');
+    my $systemd_tasks_cmd = 'echo "Triggering systemd timed service $i" && systemctl start $i';
+    $systemd_tasks_cmd .= ' && systemctl mask $i' if get_var('SOFTFAIL_BSC1063638');
```

Comment:
Maybe it is too early, but shouldn't this if be the opposite?
Mask if not SOFTFAIL_BSC1063638?

Reply:
Of course! Thanks. Updated but will test